<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LCE-CogGrp1074" ignored="0" name="Test" tests="2" failures="1" timestamp="20 Nov 2020 14:59:22 GMT" time="28.446" errors="0">
  <testcase name="testBadRequest" time="20.422" classname="Hackathon_Test.APITest"/>
  <testcase name="testGoodRequest" time="28.072" classname="Hackathon_Test.APITest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Hackathon_Test.APITest.printlog(APITest.java:798)
at Hackathon_Test.APITest.apiTestPASS(APITest.java:514)
at Hackathon_Test.APITest.testGoodRequest(APITest.java:281)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGoodRequest -->
</testsuite> <!-- Test -->
